domains/Cyanic.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Cyanic's website (link to cyanic.tk)",
+  "repo": "https://github.com/Cyanic76/Cyanic76",
+  "owner": {
+    "username": "Cyanic76",
+    "email": "git@cyanic.tk"
+  },
+  "record": {
+    "URL": "cyanic.tk"
```

Comment:
# CHOOSE ONE.

```suggestion
    "URL": "https://cyanic.tk"
```
if you want a redirect to cyanic.tk

```suggestion
    "CNAME": "cyanic.tk"
```
if you want a masked website.

Reply:
I'm sorry about it. I added the https:// protocol.